components/service/glean/src/main/java/mozilla/components/service/glean/firstrun/FirstRunDetector.kt

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.glean.firstrun
+
+/**
+ * Defines the interface for a service detecting if this
+ * is the first time the glean is used by the app.
+ */
+internal interface FirstRunDetector {
```

Comment:
Are you expecting this to have different implementations? I wonder if this level of abstraction is needed?

Reply:
Mh, this is a good point. Not right now, I'd say. Ideally, this could use some other backend implementation (shared configuration file? sqlite due to other reasons?). But not now. Maybe I should just remove the interface and reconsider later.

Reply:
Yeah, depends on how you intent to use it. Every class/interface adds a little bit to the APK size.

Reply:
I see, good point. I removed it, there's no need for it to be here now.